.eslintrc.js

```diff
@@ -275,6 +275,9 @@ module.exports = {
             allowNumber: true,
           },
         ],
+        // This rule temporarily applies the latest `@typescript-eslint/no-redundant-type-constituents` config found in `@metamask/eslint-config`.
+        // TODO: Remove once `@metamask/eslint-config` is updated to `^14.0.0`.
+        '@typescript-eslint/no-redundant-type-constituents': 'off',
```

Comment:
Sorry I am a bit confused; is this rule currently enabled, but it's being disabled in this PR? If it's being disabled, why are these new ESLint disable comments needed?

Reply:
My bad, it was somehow captured. I'll close this PR now.